app/modules/github.py

```
@@ -0,0 +1,70 @@
+from . import models
+
+
+class Github:
+    def __init__(self):
+        self.repo = "Super-Protocol/sp-vm"
+        self.latest_release_url = f"https://api.github.com/repos/{self.repo}/releases/latest"
+        self.logger = logging.getLogger(__name__)
+
+    def _get_release_json(self):
+        self.logger.info(f'fetching release json')
+        r = requests.get(self.latest_release_url)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)